client/java-armeria/src/main/java/com/linecorp/centraldogma/internal/client/armeria/ArmeriaCentralDogma.java

```diff
@@ -1031,9 +1073,11 @@ private static String toString(AggregatedHttpResponse res) {
         return res.content(charset);
     }
 
-    private static <T> Entry<T> toEntry(Revision revision, JsonNode node, QueryType queryType) {
+    private static <T> Entry<T> toEntry(JsonNode node, QueryType queryType) {
         final String entryPath = getField(node, "path").asText();
         final EntryType receivedEntryType = EntryType.valueOf(getField(node, "type").asText());
+        final Revision revision = new Revision(getField(node, "revision").asInt());
```

Comment:
The client may talk to an older Central Dogma instance. Can you make it handle an old server response as well and add a test case for it?

Reply:
Good point.

Reply:
It seems like @ikhoon just moved this logic into this method:
https://github.com/line/centraldogma/blob/e072bcc80b0977811a0349a01a12ef5ff88c954d/common/src/main/java/com/linecorp/centraldogma/internal/api/v1/EntryDto.java#L56-L59
https://github.com/line/centraldogma/blob/584f3766d6facd0a77693a65b9e2fd9e49c0b212/client/java-armeria/src/main/java/com/linecorp/centraldogma/internal/client/armeria/ArmeriaCentralDogma.java#L868

Reply:
The entry node always has the revision so it shouldn't be a problem?